_source/_data/authors.yml

```diff
@@ -493,7 +493,8 @@ jimena-garbarino:
   full_name: Jimena Garbarino
   display_name: Jimena Garbarino
   avatar: avatar-jimena.png
-  twitter: https://twitter.com/oktadev
+  twitter: https://twitter.com/indiepopart
+  github: https://github.com/indiepopart
```

Comment:
@indiepopart Are you OK with us expanding your profile? I'm happy to revert if you'd like to remain more anonymous.